src/StrawberryfieldHydroponicsService.php

```diff
@@ -247,16 +415,18 @@ public function stop() {
         $this->logger->info('Hydroponics Service could not stop because of @code', [
             '@code' => posix_strerror($errorcode)]
         );
-
         return posix_strerror($errorcode);
       } else {
         \Drupal::state()->set('hydroponics.queurunner_last_pid', 0);
         sleep(1);
         return "Successfully Stopped. Thanks";
       }
     }
+    //PID to clear, not running and not in ptable
+    else {
+      \Drupal::state()->set('hydroponics.queurunner_last_pid', 0);
+      sleep(1);
+      return "Successfully cleared pid. Thanks";
```

Comment:
This message and the way we return messages here is something I'm not totally convinced of
1.- Messages are not translated. 
2.- We depend on a message to define if stopped worked? DO we need a also a status returned? We could use maybe instead a StdClass Object to make sending/returning of data easier like we do for the actual workers?

Reply:
@DiegoPino As for checkRunning, stop function can return an array ['staus'] (TRUE all ok, FALSE for error) and ['message'] with text about result, what do you think?